spaces/domains.json

```
@@ -378,5 +378,6 @@
   "voting-v2.multiplier.finance": "v2-multiplierfinance.eth",
   "vote.mgh.dao": "metagamehub.eth",
   "gov.maidcoin.org": "maidcoin.eth",
-  "vote.cyberkongz.com": "cyberkongz.eth"
+  "vote.cyberkongz.com": "cyberkongz.eth",
+  "dao.baka.cool":"koumakan.eth"
```

Please add a space here.
```suggestion
  "dao.baka.cool": "koumakan.eth"
```

---

resolve